docs/Advanced-Usage/Generating-Different-Targets.rst

```diff
@@ -101,13 +101,49 @@ compound Config instance. See the example below:
     class Override2 extends Config((site, here, up) => {...})
     // Specify by setting TARGET_CONFIG=Compound
     class Compound extends Config(new Override2 ++ new Override1 ++ new Base)
-    // OR by setting TARGET_CONFIG=Base_Override1_Override2
-    // Can specify undefined classes this way. ex: TARGET_CONFIG=Base_Override2
+    // OR by setting TARGET_CONFIG=Override2_Override1_Base
+    // Can specify undefined classes this way. ex: TARGET_CONFIG=Override2_Base
 
 With this scheme, you don't need to define a Config class for every instance you
 wish to generate. We use this scheme to specify FPGA frequencies (eg.
-BaseF1Config_F90MHz) in manager build recipes, but it's also very useful for doing
-sweeping over a parameterization space.
+"BaseF1Config_F90MHz") in manager build recipes, but it's also very useful for doing
+sweeping over a parameterization space. 
+
+**Note that the precedence of Configs decreases from left to right in a string**.  Appending a config to an existing one will only have an effect if it
```

Comment:
```suggestion
**Note that the precedence of Configs decreases from left to right in a string**.  Appending (as opposed to prepending) a config to an existing one will only have an effect if it
```

Reply:
I think this is unambiguous enough, given that you've said it in other places, and there's an example below. The docs are too wordy as it is.